attributes/network/9_account_features/cddl/1_account_multisig.cddl

```diff
@@ -38,6 +38,13 @@ event-type /= account-event-type-multisig-submit
            /  account-event-type-multisig-set-defaults
            /  account-event-type-multisig-expire
 
+; Can filter events on multisig-transaction-state.
+events-filter-attribute-specific /= {
+    ; Filter a multisig event by its type. This is only valid for multisig
+    ; event types, otherwise should not match any event.
+    ? [ 9, [ 1, 0 ] ] = multisig-transaction-state / [ * multisig-transaction-state ]
```

Comment:
Can we do that in `minicbor`?

From the `minicbor` doc, it is written
> The encoding does not contain any names, i.e. no field names, type names or variant names. Instead, every field and every constructor needs to be annotated with an (unsigned) index number, e.g. #[n(1)].

Reply:
We will need to implement our own `Encode` / `Decode`.